ci-operator/config/openshift/sriov-cni/openshift-sriov-cni-release-4.15.yaml

```
@@ -10,7 +10,7 @@ base_images:
 build_root:
   from_repository: true
 images:
-- dockerfile_path: Dockerfile.rhel7
+- dockerfile_path: Dockerfile.rhel
   from: base
   inputs:
     ocp_builder_rhel-8-golang-1.20-openshift-4.14:
```

Can you also updated the input image

```suggestion
    ocp_builder_rhel-9-golang-1.20-openshift-4.14:
      as:
      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.14
```